getCalspec/rebuild.py

```diff
@@ -43,9 +45,24 @@ def add_astroquery_id(df):
 
 
 def add_alt_star_name(df):
+    name_columns = [name for name in df.columns if "name" in name.lower()]
+    print(name_columns)
     for i, row in df.iterrows():
         if row["Star name"] == "ETA1 DOR":
             df.at[i, "Alt Star name"] = "ETA DOR"
+        else:
+            all_names = None
+            for name in name_columns:
+                all_names = Simbad.query_objectids(row[name])
+                if all_names is not None and len(all_names) > 0:
+                    break
+            if all_names is not None:
+                for name in list(all_names['ID']):
+                    if "HD" in name:
+                        df.at[i, "Alt Star name"] = name.replace(' ','')
+            else:
+                continue
```

Comment:
I don't _think_ these lines actually do anything, right? You're continuing at what is already the end of the loop...